src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,78 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
 public class WorkWithFile {
+    private static final String SUPPLY_CONDITION = "supply";
+    private static final String BUY_CONDITION = "buy";
+    private static final String REGEX_CONDITION = "\\D";
+    private StringBuilder stringBuilder = new StringBuilder();
```

Comment:
this is not a good practice to store state in service classes
so let's better move this variable to method